unit_tests/test_ycsb_thread.py

```diff
@@ -10,7 +10,8 @@
 
 from unit_tests.dummy_remote import LocalLoaderSetDummy
 
-pytestmark = pytest.mark.usefixtures('events', 'create_table')
+pytestmark = [pytest.mark.usefixtures('events', 'create_table'),
```

Comment:
how does this work?

Reply:
why not to use `pytest.mark.skip` on the test?

Reply:
http://doc.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules